src/main/kotlin/com/github/weusearchbtw/ccc_2023_oct/lib/Differ.kt

```diff
@@ -41,7 +34,7 @@ object Differ {
 		// fallback for non-linux systems
 		val out = ByteArrayOutputStream()
 		val rt1 = RawText(expected.toByteArray(Charsets.UTF_8) + NEWLINE.toByte())
-		val rt2 = RawText(output.toByteArray(Charsets.UTF_8) + NEWLINE.toByte())
+		val rt2 = RawText(outputFile.toByteArray(Charsets.UTF_8) + NEWLINE.toByte())
```

Comment:
expected* so isses der dateiname

Reply:
kana braucht an fallback für non linux. Linux is superior